parser/ast/dml.go

```diff
@@ -3667,10 +3667,12 @@ func (n *AsOfClause) Accept(v Visitor) (Node, bool) {
 		return v.Leave(newNode)
 	}
 	n = newNode.(*AsOfClause)
-	node, ok := n.TsExpr.Accept(v)
-	if !ok {
-		return n, false
+	if n.TsExpr != nil {
```

Comment:
It's more robust. But I'm just curious any time `n.TsExpr` is nil but `*AsOfClause` is present?

Reply:
Just for test, it makes it easier to construct a `AsOfClause`, and test the visitor :facepalm: